api/utils.py

```diff
@@ -3,53 +3,59 @@
 from urllib.request import Request, urlopen
 
 import orjson
+import i18n
 
+i18n.set("file-format", "yaml")
+i18n.set("filename_format", "{locale}.{format}")
+i18n.set("enable_memoization", True)
+i18n.load_path.append("./api/locale")
```

Comment:
i18n [currently](https://github.com/danhper/python-i18n/blob/master/i18n/config.py) has a default `"fallback`" set to `"en"`, meaning if an error occurs in translation for any reason, it can default to checking `en.yml`.

Since there is no `en.yml`, it may be a good idea to either change `en-us.yml` to `en.yml` and make that the default everywhere, or set the fallback to `en-us` in the configuration.

I'm fine with it either way, feel free to let me know what you think.

It also seems `yml` is the default if available, so might not need to set the `file_format` to `yaml`, and if it is set, it should probably be `yml` rather than `yaml`. It also seems like it should use an underscore?